tests/st/calicoctl/test_default_pools.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2015-2016 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import netaddr
+import yaml
+from nose_parameterized import parameterized
+
+from tests.st.test_base import TestBase
+from tests.st.utils.docker_host import DockerHost
+from tests.st.utils.exceptions import CommandExecError
+from tests.st.utils.utils import ETCD_CA, ETCD_CERT, \
+    ETCD_KEY, ETCD_HOSTNAME_SSL, ETCD_SCHEME, get_ip, wipe_etcd, retry_until_success
+
+# Load the calico-node we've just built
+POST_DOCKER_COMMANDS = ["docker load -i /code/calico-node.tar"]
+if ETCD_SCHEME == "https":
+    ADDITIONAL_DOCKER_OPTIONS = "--cluster-store=etcd://%s:2379 " \
+                                "--cluster-store-opt kv.cacertfile=%s " \
+                                "--cluster-store-opt kv.certfile=%s " \
+                                "--cluster-store-opt kv.keyfile=%s " % \
+                                (ETCD_HOSTNAME_SSL, ETCD_CA, ETCD_CERT,
+                                 ETCD_KEY)
+else:
+    ADDITIONAL_DOCKER_OPTIONS = "--cluster-store=etcd://%s:2379 " % \
+                                get_ip()
+
+
+class TestDefaultPools(TestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        # First, create a host to run things in
+        cls.hosts = []
+        cls.host = DockerHost("host",
+                              additional_docker_options=ADDITIONAL_DOCKER_OPTIONS,
+                              post_docker_commands=POST_DOCKER_COMMANDS,
+                              start_calico=False)
+        cls.hosts.append(cls.host)
+
+    def setUp(self):
+        try:
+            self.host.execute("docker rm -f calico-node")
+        except CommandExecError:
+            # Presumably calico-node wasn't running
+            pass
+        wipe_etcd(get_ip())
+
+    @classmethod
+    def tearDownClass(cls):
+        for host in cls.hosts:
+            host.cleanup()
+
+    @parameterized.expand([
+        (False, "CALICO_IPV4POOL_CIDR", "10.0.0.0/27",  None),    # Too small
```

Comment:
Please move the informative descriptions from comment into a parameter.  That way, you can log them and they'll probably get spat out by `parametrised.expand` too.